crates/subspace-farmer/src/bin/subspace-farmer/commands/bench.rs

```diff
@@ -54,9 +55,9 @@ impl PlotFile for BenchPlotMock {
     }
 }
 
-struct HumanReadable(pub u64);
+struct HumanReadable<T>(pub T);
```

Comment:
Please don't do this. It is confusing to combine size and duration in the same struct for different types, especially when one of them is as generic as `u64`.

I'd do this instead:
```suggestion
struct HumanReadableDuration(Duration);
```

And similarly renamed for sizes too.